validations/semantic_uniqueness.go

```diff
@@ -18,41 +19,41 @@ func isNameUnique(mta *mta.MTA, mtaNode *yaml.Node, source string, strict bool)
 	// map: name -> object kind (module, provided services or resource)
 	names := make(map[string]nameInfo)
 	for i, module := range mta.Modules {
-		line := getModuleLineByIndex(mtaNode, i)
+		line, column := getModuleLineByIndex(mtaNode, i)
 		// validate module name
-		issues = validateNameUniqueness(names, module.Name, moduleEntityKind, issues, line)
+		issues = validateNameUniqueness(names, module.Name, moduleEntityKind, issues, line, column)
 		for j, provide := range module.Provides {
-			setLine := getProvidedSetByIndex(mtaNode, i, j)
+			setLine, setColumn := getProvidedSetByIndex(mtaNode, i, j)
```

Comment:
also may  be getProvidedSetPositionByIndex?

Reply:
Done